src/jAlert.js

```diff
@@ -137,10 +137,12 @@
                         if(typeof onClose == 'function')
                         {
                             onClose(alert.instance);
+                            toggleFocusTrap();
                         }
                         else if(typeof alert.onClose == 'function')
                         {
                             alert.onClose(alert.instance);
+                            toggleFocusTrap();
                         }
```

Comment:
You should call toggleFocusTrap() regardless of whether or not there is another onClose. You can remove these and put it above or below this if elseif